### PR TITLE
Improve the design the individual movie page

### PR DESCRIPTION
**Pull request by z3t1**
Do not merge yet. 

screen shot 2015-06-24 at 10 13 58 pm

The rightmost column about movie's basic info is not working. I used nested `{{#each}}` of handlebars. It doesn't give any result.

### CODE REVIEW

**Comment by vergenzt** [comment]
http://stackoverflow.com/a/14805966

File: views/movie.handlebars [hunk 1]
```diff
@@ -1,30 +1,74 @@
...
+      {{#each abridged_directors}}
+        <span>{{name}}&nbsp;</span>
+      {{/each}}
+    </p>
+    <p>Cast:
+      {{#each abridged_cast}}
+        {{name}}&nbsp;
+      {{/each}}
+    </p>
+    <p>Genres: {{#each genres}}{{this}}{{/each}}</p>
```
Comment (vergenzt) [inline views/movie.handlebars]: `{{#each movie.genres}}`

Comment (vergenzt) [inline views/movie.handlebars]: I think

**Comment by z3t1** [comment]
@vergenzt It works. That wasn't nested each each actually. There isn't a top level {{#each movies}}. I messed it up with movies.handlebars.

**Pull request merged**